mailbox/api/src/test/java/org/apache/james/mailbox/model/AttachmentTest.java

```diff
@@ -31,43 +32,45 @@
 
 public class AttachmentTest {
 
+    private static Charset CHARSET = Charset.defaultCharset();
```

Comment:
Because you want your test to be reproducible. And if you don't set a charset, the default charset can be different depending on the computer you run the test on.
That's the same reason why we set the date and don't use Date.now() in tests.